vendor/rakwireless/rak7431-868-profile.yaml

```diff
@@ -0,0 +1,17 @@
+vendorProfileID: 7431
```

Comment:
Does RAK have different LoRaWAN Alliance Vendor Profile IDs? In the index, it looks like your VendorID is 570: https://github.com/TheThingsNetwork/lorawan-devices/blob/master/vendor/index.yaml#L904

Reply:
Thanks, your description suggests this was our own number
"# Vendor profile ID, can be freely issued by the vendor"
So is this actually our LoRaWAN alliance ID? Not device specific?

Reply:
This is device profile specific, so it looks good to me.